application/main.py

```diff
@@ -573,7 +573,85 @@ def __eq__(self, other):
                (self.validation_results == other.validation_results)
 
 
+@application.route('/build_implementer_agreement_results/<id>')
```

Comment:
This is similar to `@application.route('/build_informal_propositions_results/<id>')` . Can you please make only one endpoint that will return both the IP and IA results?

Reply:
Makes sense. Changed it a little bit, now I propose to use `build_results `endpoint. It's results:

```
[{"lineno": null, "results": [{"entity_id": 844, "entity_type": "IfcAlignmentHorizontal", "msg": "Rule passed", "status": "v"}, {"entity_id": 832, "entity_type": "IfcAlignmentHorizontal", "msg": "Rule passed", "status": "v"}],
  "step_type": "implementer_agreement", "text": "Each IfcAlignmentHorizontal must nest only 1 instance(s) of IfcAlignment"},
 {"lineno": null, "results": [{"entity_id": 831, "entity_type": "IfcAlignment", "msg": "Rule passed", "status": "v"}, {"entity_id": 830, "entity_type": "IfcAlignment", "msg": "Rule passed", "status": "v"}],
  "step_type": "implementer_agreement", "text": "Each IfcAlignment must be nested by at most 1 instance(s) of IfcAlignmentVertical"}, {"lineno": null, "results": [
    {"entity_id": 820, "entity_type": "IfcAlignmentHorizontalSegment", "msg": "Rule passed", "status": "v"}, {"entity_id": 809, "entity_type": "IfcAlignmentHorizontalSegment", "msg": "Rule passed", "status": "v"},
    {"entity_id": 835, "entity_type": "IfcAlignmentHorizontalSegment", "msg": "Rule passed", "status": "v"}, {"entity_id": 821, "entity_type": "IfcAlignmentHorizontalSegment", "msg": "Rule passed", "status": "v"}],
                                                                                                                                       "step_type": "implementer_agreement",
                                                                                                                                       "text": "The type of attribute DesignParameters should be IfcAlignmentHorizontalSegment"},
 {"lineno": null, "results": [{"entity_id": null, "entity_type": null, "msg": "On instance #425=IfcShapeRepresentati...(#274)) the following invalid value for RepresentationType has been found: Curve3D", "status": "i"},
                              {"entity_id": null, "entity_type": null, "msg": "On instance #193=IfcShapeRepresentati...(#183)) the following invalid value for RepresentationType has been found: Segment", "status": "i"},
                              {"entity_id": null, "entity_type": null, "msg": "On instance #882=IfcShapeRepresentati...(#872)) the following invalid value for RepresentationType has been found: Segment", "status": "i"},
                              {"entity_id": null, "entity_type": null, "msg": "On instance #893=IfcShapeRepresentati...(#883)) the following invalid value for RepresentationType has been found: Segment", "status": "i"}],
  "step_type": "informal_proposition", "text": "The values must be in 'valid_RepresentationType.csv'"}]
```

So the change in the json is the addition of `step_type `attribute, which specifies if the results come from `informal_proposition `or `implementer_agreement`.